hello_vsi/README.md

```diff
@@ -1,32 +1,68 @@
 # Hello VSI example
 
-This example uses the VSI ([Virtual Streaming Interface](https://arm-software.github.io/AVH/main/simulation/html/group__arm__vsi.html)) to simulate a peripheral sensor. The example sensor will send data in form of bytes to the application running on [AVH FVP Models](https://arm-software.github.io/AVH/main/overview/html/index.html). The virtual peripheral is controlled by a python script that reads the data from a text file called `intdata.txt`. This is a minimal example of how to use Arm VSI, and you can modify the python script and the sensor driver in order to match your use case.
+This example uses the **VSI** ([Virtual Streaming Interface](https://arm-software.github.io/AVH/main/simulation/html/group__arm__vsi.html)) to simulate a peripheral sensor. The example sensor will send data in form of bytes to the application running on [AVH FVP Models](https://arm-software.github.io/AVH/main/overview/html/index.html). The virtual peripheral is controlled by a python script that reads the data from a text file called `intdata.txt`. This is a minimal example of how to use Arm VSI, and you can modify the python script and the sensor driver in order to match your use case.
 
 ## Build and run
 
-Use the cbuild tool or an IDE to build the application project in csolution format (See the project [README](../README.md).
+Use the cbuild tool or an IDE to build the application project in csolution format (see the project [README](../README.md)).
 
-All applications have been ported for Corstone-300, Corstone-310 and Corstone-315, and can be compiled with Arm Compiler 6 or GCC. By default the Arm Compiler 6 is used, to compile with GCC, use `--toolchain GCC` option for cbuild command.
+All applications have been ported for **Corstone-300**, **Corstone-310** and **Corstone-315**, and can be compiled with **Arm Compiler** 6 or **GCC**. By default the Arm Compiler 6 is used, to compile with GCC, use `--toolchain GCC` option for the cbuild command.
```

Comment:
To make this "symmetric" you could choose not to specify a `compiler:` node in the `csolution.yml` file and use `--toolchain AC6` and `--toolchain GCC` at the command line instead?

Reply:
Has been corrected as follow:

All applications have been ported for **Corstone-300**, **Corstone-310** and **Corstone-315**, and can be compiled with **Arm Compiler 6** or **GCC**. To compile with **AC6** use `--toolchain AC6` and for **GCC** the `--toolchain GCC` option with the cbuild command.